src/Mod/Sketcher/Gui/ViewProviderSketch.h

```diff
@@ -407,6 +408,9 @@ class SketcherGuiExport ViewProviderSketch : public PartGui::ViewProvider2DObjec
     App::PropertyString EditingWorkbench;
     //@}
 
+    static SbVec2s prvMoveCursorPos;
+    Gui::View3DInventorViewer * prvMoveViewer;

```

Comment:
I made these two because I want to trigger a mouseMove of the SketchHandler from within the ToolSettings Widget to update the tool when something is typed in.
See TaskSketcherTool.cpp line 141 : 
sketchView->mouseMove(sketchView->prvMoveCursorPos, sketchView->prvMoveViewer);
It felt so bad doing this but I was at a loss. It's linked to the previous thing that the widget class is declared in VP.